v3/integration/lints/lints/init_first_test.go

```diff
@@ -0,0 +1,47 @@
+package lints
+
+/*
+ * ZLint Copyright 2021 Regents of the University of Michigan
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+import (
+	"os"
+	"testing"
+
+	"github.com/zmap/zlint/v3/integration/lints/lint"
+)
+
+func TestInitFirst_Lint(t *testing.T) {
+	t.Log(os.Getwd())
```

Comment:
Was this leftover from debugging or something you think is useful to keep long term?